backend/algorithms/objects/course.py

```diff
@@ -11,12 +11,12 @@ class Course:
     mark: int
     uoc: int
     terms: dict[int, list[int]]
-    def term_domain(self, start: Tuple[int, int], end: Tuple[int, int]):
+    def term_domain(self, start: Tuple[int, int], end: Tuple[int, int]) -> List[List[int]]:
         """ create a domain of terms this course can be in for autoplanning """
-        numbers = []
-        for key, value in self.terms.items():
-            for term in value:
-                new_number = (key - start[0]) * 4 + term - start[1]
-                if new_number <= (end[0] - start[0]) * 4 + end[1] - start[1]:
-                    numbers.append(new_number)
+        numbers = [
```

Comment:
HAHAHHAHA I tried making this comprehension when I was writing this, I'm glad it's possible to do